src/templates/static/legal/regulation.jsx

```diff
@@ -202,12 +202,19 @@ const Regulation = () => {
                 <p>{it.L('[_1] is not available in Costa Rica and is not regulated or supervised by any regulatory body in the Costa Rican financial system.', it.website_name)}</p>
             </Box>
 
-            <Box header={it.L('Binary (V) Ltd')} last >
+            <Box header={it.L('Binary (V) Ltd')} >
                 <p>{it.L('Binary (V) Ltd, Govant Building, Port Vila, PO Box 1276, Vanuatu, Republic of Vanuatu. Licensed and regulated by the Vanuatu Financial Services Commission - <a href="https://www.vfsc.vu/wp-content/uploads/2015/12/List-of-Licensees-under-Dealers-in-Securities-Licensing-Act-CAP-70-18.11.2016.pdf" target="_blank">view licence</a>.')}</p>
                 <p>{it.L('Clients from the rest of the world (excluding certain countries such as the USA, Costa Rica, Hong Kong) will have their FX and CFD Metatrader 5 account opened with Binary (V) Ltd. [_1] is not available in the Republic of Vanuatu.', it.website_name)}</p>
                 <RegulatorText />
                 <RegulatorImage href='https://www.vfsc.vu/' image='images/pages/regulation/vanuatu-logo.svg' />
             </Box>
+            
+            <Box header={it.L('Binary (BVI) Ltd')} last>
+                <p>{it.L('Binary (BVI) Ltd, 2nd Floor, O’Neal Marketing Associates Building, Wickham’s Cay II, P.O. Box 3174, Road Town, Tortola VB1110, British Virgin Islands. Licensed and regulated by the British Virgin Islands Financial Services Commission - <a href="http://www.bvifsc.vg/en-us/regulatedentities/investmentbusiness/categoriesofinvestmentbusinesslicences/category1dealingininvestments/subcategoryadealingininvestments.aspx" target="_blank">view licence</a>.')}</p>
```

Comment:
please move the link out of the translation string and use a place holder instead

Reply:
ok, sure